autoptsserver.py

```diff
@@ -407,7 +406,7 @@ def parse_args(self, args=None, namespace=None):
 
 
 def get_workspace(workspace):
-    for root, dirs, files in os.walk(os.path.join(PROJECT_DIR, 'autopts/workspaces'),
+    for root, dirs, _files in os.walk(os.path.join(PROJECT_DIR, 'autopts/workspaces'),
```

Comment:
I guess this is because `files` were unused? 
I'm a bit rusty in my Python/PEP8 :D